src/utils/gitHubUtils.ts

```diff
@@ -103,9 +103,19 @@ export async function getGitHubQuickPicksWithLoadMore<TResult, TParams>(
     }
 }
 
+const githubProviderId: string = 'github';
+const scopes: string[] = ['repo', 'workflow', 'admin:public_key'];
+
 export async function getGitHubAccessToken(): Promise<string> {
-    const scopes: string[] = ['repo', 'workflow', 'admin:public_key'];
-    return (await authentication.getSession('github', scopes, { createIfNone: true })).accessToken;
+    return (await authentication.getSession(githubProviderId, scopes, { createIfNone: true })).accessToken;
+}
+
+export async function logoutOfGitHubSession(): Promise<void> {
+    const sessionId: string | undefined = (await authentication.getSession(githubProviderId, scopes))?.id;
+
+    if (sessionId) {
+        await authentication.logout(githubProviderId, sessionId);
```

Comment:
I don't see logout in the official API and I thought we were supposed to have moved off of the proposed API a long time ago

Reply:
Yeah, you're right.  It's actually deprecated in the proposed api... 😬